torchvision/io/video.py

```diff
@@ -197,14 +204,19 @@ def read_video(filename, start_pts=0, end_pts=None):
 
     vframes = [frame.to_rgb().to_ndarray() for frame in video_frames]
     aframes = [frame.to_ndarray() for frame in audio_frames]
-    vframes = torch.as_tensor(np.stack(vframes))
+
     if aframes:
         aframes = np.concatenate(aframes, 1)
         aframes = torch.as_tensor(aframes)
         aframes = _align_audio_frames(aframes, audio_frames, start_pts, end_pts)
     else:
         aframes = torch.empty((1, 0), dtype=torch.float32)
 
+    if vframes:
+        vframes = torch.as_tensor(np.stack(vframes))
+    else:
+        vframes = torch.empty((1, 0), dtype=torch.float32)
```

Comment:
The type of the video frames are `uint8` by default, and should have 4 channels. Can you do instead
```python
vframes = torch.empty((0, 1, 1, 3), dtype=torch.uint8)
```
or something like that? We should have a meaningful default for the empty case, so we should decide if it would be a height of 1 or 0 (or something else)